README.md

```diff
@@ -315,8 +315,8 @@ Returns: `Dispatcher`
 
 ## Specification Compliance
 
-This section documents parts of the HTTP/1.1 specification which Undici does
-not support or does not fully implement.
+This section documents parts of the HTTP/1.1 specification that Undici does
+not support or fully implement.
```

Comment:
IMHO incorrect

Reply:
Updated in https://github.com/nodejs/undici/pull/1354/commits/4d9f60dd622662e6fc11b26a19d480cf17b4fc53